assets/css/components/_pagination.scss

```diff
@@ -11,40 +11,63 @@
 
   .prev-next {
     height: rem(53);
-    margin: 0 rem(20);
+    margin: 0 rem(6);
   }
 
   li {
     display: inline;
     text-align: center;
 
+    .pagination-ellipsis {
```

Comment:
The ellipsis has hover/focus styles, which it shouldn't as it's not interactive. I thought I remembered us discussing that it should be styled as plain text?

Reply:
These have been addressed in the latest push